app/controllers/coronavirus_form/medical_equipment_controller.rb

```diff
@@ -4,26 +4,27 @@ class CoronavirusForm::MedicalEquipmentController < ApplicationController
   skip_before_action :check_first_question_answered
 
   def submit
-    medical_equipment = strip_tags(params[:medical_equipment]).presence
-
-    session[:medical_equipment] = medical_equipment
+    @form_responses = {
+      medical_equipment: strip_tags(params[:medical_equipment]).presence,
+    }
 
     invalid_fields = validate_radio_field(
       controller_name,
-      radio: medical_equipment,
+      radio: @form_responses[:medical_equipment],
     )
 
     if invalid_fields.any?
       flash.now[:validation] = invalid_fields
       log_validation_error(invalid_fields)
       render controller_path
     elsif session["check_answers_seen"]
+      session[:medical_equipment] = @form_responses[:medical_equipment]
       redirect_to check_your_answers_url
-    elsif session[:medical_equipment] == I18n.t("coronavirus_form.questions.#{controller_name}.options.option_yes.label")
-      # YES goes to what kind of business are you / are_you_a_manufacturer
+    elsif @form_responses[:medical_equipment] == I18n.t("coronavirus_form.questions.#{controller_name}.options.option_yes.label")
+      session[:medical_equipment] = @form_responses[:medical_equipment]
       redirect_to are_you_a_manufacturer_url
-    elsif session[:medical_equipment] == I18n.t("coronavirus_form.questions.#{controller_name}.options.option_no.label")
-      # NO goes to hotel rooms
+    elsif @form_responses[:medical_equipment] == I18n.t("coronavirus_form.questions.#{controller_name}.options.option_no.label")
+      session[:medical_equipment] = @form_responses[:medical_equipment]
       redirect_to hotel_rooms_url
     else
       render controller_path
```

Comment:
This is out of scope for this PR, but why is this `else` option here?

Reply:
I can't see any use for it.  Have added an additional commit to remove.